resources/arduino_files/camera_board/camera_board.ino

```diff
@@ -152,7 +125,7 @@ void setup(void) {
   turn_pin_on(CAM_1_RELAY);
 
   acc_handler.init();
-  dht_handler.init();
+  dht_handler.Init();
```

Comment:
Is the plan to make the shared libraries use the `Init` vs `init`?

Reply:
Yes, if you don't mind. I'm comfortable with the Google C++ Style Guide, where accessors are lower case (e.g. `float humidity() const;` or `void set_humidity(float v);`, and other methods are upper case (e.g. `float ComputeSomeValue();`).

Reply:
Yeah, that's fine and good to know. I'm not familiar with good c++ coding styles.